more meatballs

wxtim opened the pull request:
It's good practice to insert meaningful information here. Please imagine that I have done so.

wxtim committed:
more meatballs

alanjhewitt commented on forecast.md:
```diff
@@ -7,3 +7,4 @@ Shitty.
 ## Tomorrow
 
 Still shitty but with a chance of meatballs
+MMM.....................meatballs.
```
@wxtim  for future reference, i clicked on the plus symbol in the diff to make a comment on this line number
```suggestion
MMM.....................meatballs!
```

alanjhewitt reviewed (changes requested):
one change for you to make

wxtim committed:
exclamation mark

Don't use actual exclamation marks in commit messages - it can mess up shell scripts later.

Co-authored-by: Alan J Hewitt <40797340+alanjhewitt@users.noreply.github.com>

alanjhewitt reviewed (approved):
looks ok
